sale_financial_risk/models/sale.py

```diff
@@ -68,7 +68,7 @@ def _compute_amount_to_invoice(self):
         """
         for line in self.filtered(lambda l: l.state == 'sale'):
             invoice_lines = line.invoice_lines.filtered(
-                lambda l: l.invoice_id.state in {'open', 'in_payment', 'paid'})
+                lambda l: l.invoice_id.state != 'cancel')
```

Comment:
Are you including `draft` invoices then?

Reply:
Yes. Updated PR description.